pyproject.toml

```diff
@@ -1,3 +1,65 @@
+[project]
+name = "pgcli"
+authors = [{ name = "Pgcli Core Team", email = "pgcli-dev@googlegroups.com" }]
+license = { text = "BSD" }
+description = "CLI for Postgres Database. With auto-completion and syntax highlighting."
+readme = "README.rst"
+classifiers = [
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: Unix",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: SQL",
+    "Topic :: Database",
+    "Topic :: Database :: Front-Ends",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+urls = { Homepage = "http://pgcli.com" }
+requires-python = ">=3.8"
+dependencies = [
+    "pgspecial>=2.0.0",
+    "click >= 4.1",
+    "Pygments>=2.0",
+    "prompt_toolkit>=2.0.6,<4.0.0",
```

Comment:
There was a comment about the pinning of "prompt_toolkit". If its removal is intentional, I'd rather see it in a distinct commit (which would explain why), that would be clearer.

Reply:
Sorry. Fixed as well.